Updating documentation to include helm 2 to helm 3 chart migration (#1231)
scality/Zenko: docs/docsource/installation/upgrade/upgrade_zenko.rst

      Expect the cluster to take a few minutes to stabilize. You may see 
      CrashLoopBackoff errors. This is normal, expected behavior.

> #. With the introduction of Helm 3 support of Helm 2 charts are still 
>    supported. Helm 3 charts introduces improved functionality with support for 
>    existing Helm 2 charts and in place upgrades and conversions.

Reviewer: ```suggestion
With the introduction of Helm 3, Helm 2 charts are still 
supported. Helm 3 charts introduce improved functionality, and support
existing Helm 2 charts and in-place upgrades and conversion.
```

Reviewer: This is not a continuation of the previous process. It introduces a new process (numbered, below)

>    To upgrade existing Helm 2 charts to Helm 3 you can install the 2to3
>    plugin and use the following steps to migrate and cleanup your 

Reviewer: ```suggestion
   To upgrade existing Helm 2 charts to Helm 3, install the 2to3
   plugin and use the following steps to migrate and clean up your 
```

   Helm 2 configuration, releases, and Tiller deployment. 

>    First install the 2to3 plugin for Helm 3.

Reviewer: ```suggestion
#. Install the 2to3 plugin for Helm 3.
```

  
  ::

     $ helm3 plugin install https://github.com/helm/helm-2to3

>   Then update the existing Helm 2 configuration.  As previously noted always 
>   test with the --dry-run option before migrating or updating production 
>   systems. 

Reviewer: ```suggestion
#. Update the existing Helm 2 configuration.  Always 
   test with the ``--dry-run`` option before migrating or updating a production system.
```

  ::

     $ helm3 2to3 move config --dry-run
     $ helm3 2to3 move config

>   After the Helm 3 configuration is complete you can convert your existing
>   release with the following steps. 

Reviewer: ```suggestion
#. After Helm 3 configuration is complete, convert the existing
    deployment with the following steps:
```
